docs/docs/intro.md

```diff
@@ -1,192 +1,79 @@
-# postgres_config contains the credentisla of datasource that we want to connect
-# for the simplicity we are using the same the database that we are using to store 
-# all inspektor metadata.
-postgres_config:
-  target_addr: "localhost"
-  target_port: "5432"
-  target_username: "postgres"
-  target_password: "postgrespass"
-  proxy_listen_port: "8081"
-```
-
-Run the below command to run the dataplane.
-
-```sh
-docker run -v $(pwd)/dataplane_config.yaml:/dataplane_config.yaml --network=host -e RUST_LOG=inspektor=debug schoolboy/inspektor-dataplane:latest1 ./inspektor --config_file ./dataplane_config.yaml
-
-```
-## Inspektor basic features
-
-The above steps will run a Postgres instance, controlplane and dataplane. The dataplane is connected to the same Postgres container that is used by controlplane to store metadata to ease out the complexitiy. But, you can connect to any Postgres instance as you like by tinkering with the dataplane config file.
-
-In this sample database, we want to protect first_name of the actor table. For this, we have to define policy using Open Policy Agent. We have already defined that policy on [sample repo](https://github.com/poonai/inspektor-policy.git)
-
-
-Now hit the [http://localhost:3123](http://localhost:3123) and use the following credentials to login into the dashboard.
-
-```
-username: admin
-password: admin
-```
-
-After login, you'll see the list of datasources that controlplane manages. 
-
-![Dashboard](../static/img/dashboard.png)
-
-The Postgres instance that we want to enforce policy is already added as datasource, now you can click on create credentials button to get login information to access the datasource.
-
-After creating the credentials you'll get to see show credentials button. After clicking you'll get a modal showing the credentials to access the Postgres instance.
-
-![Credentials Modal](../static/img/credentials.png)
-
-Now just use psql to login to the Postgres instance using the copied credentials from the dashboard.
-
-```
-psql "sslmode=disable host=localhost port=8081 dbname=postgres user=<username>"
-```
+**Controlplane:** The control plane is the part of a network that controls how data packets are forwarded — meaning how data is sent from one place to another. The process of creating a routing table, for example, is considered part of the control plane. 
 
-After executing the above command, psql will prompt you to enter password. Enter the password
-which you copied from the modal to login.
+Routers use various protocols to identify network paths, and they store these paths in routing tables.
 
-Now that, you logged in. execute a simple select query on the actor table.
+and...
 
-```sql
-select * from actor;
-```
+**Dataplane:** In contrast to the control plane, which determines how packets should be forwarded, the data plane actually forwards the packets. The data plane is also called the forwarding plane.
```

Comment:
control plane and dataplane definitions are not resonating with our use case.

Reply:
I'll come up with some thing for this

Reply:
yeah, this 2 words were alien to me before